specification/v3.4/OSDM-online-api-v3.4.0-draft.yml

```diff
@@ -14143,7 +14143,7 @@ components:
       properties:
         departureTime:
           description: |
-            Needs to in local date time format.
+            Needs to in local date time format. If departure time is provided the departure time must be omitted.
```

Comment:
Should be
"If departure time is provided the arrival time must be omitted." to mimic line 14152